### ISSUE TITLE
Add taxonomic distribution classifier to QLT

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1041]
I really thought I made a ticket for this before, but clearly not....
(I included the numbers just for ref)

The queries are all mutually exclusive, they are the same ones that I use in the figure

272 Label : Fungi and Prokaryotes
((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) NOT ((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) AND conserved in metazoa (PBO:0011069)))

2540 Label :Eukaryotic, to Metazoa
Query: (conserved in metazoa (PBO:0011069) NOT ((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) AND conserved in metazoa (PBO:0011069)))

1009  Label : Kingdom wide, to metazoa
Query: ((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) AND conserved in metazoa (PBO:0011069))

546	Label : Fungi only
Query: conserved in fungi only (PBO:0011063)

147	Label: S. pombe specific
Query: Schizosaccharomyces pombe specific (PBO:0016878)	...

221	Label: Schizos. specific
Query Schizosaccharomyces specific (PBO:0016885)

55	Label: dubious
Query:    characterisation_status:dubious

The above are mutually exclusive with each other (or will be tomorrow)

337 (Label : Other)

((subset: feature_type_mRNA_gene NOT subset: characterisation_status_transposon) NOT subset: characterisation_status_dubious)
MINUS the ABOVE

there are mainly taxonomic conservation unclear (massively duplicated families), or fusions, horizontal transfer.
I might separate our fusions and horizontal transfer later… but it makes sense for these odd bods to be lumped for now.

I'll draft some documentation for this.

**Comment by ValWood** [comment]
Also, how could I forget this:

gene characterization (7 mutually exclusive classifiers)

https://www.pombase.org/status/gene-characterisation

(I forgot about it because I only used conserved unknown in the figure)

It will be interesting to see the differences between these.

**Comment by ValWood** [comment]
Let me know if none of these make sense. It would be nice to add soon, since I *think* it would be quite quick? Then we can announce.

**Comment by kimrutherford** [comment]
> Gene characterization should be quick.

That's live.  But now it's on the main site I realise that the label for the case where no characterisation has been assigned doesn't make sense.  It currently says "Not applicable".  Would "Not curated" be better?  This is mostly for RNAs and pseudogenes as far as I can see.

**Comment by ValWood** [comment]
Not curated would be perfect "not curated for this datatype" if space

**Comment by kimrutherford** [comment]
> "not curated for this datatype"

Yep, that's no problem.

This one is a bit long though: "Schizosaccharomyces specific protein, uncharacterized".  Would it be OK to shorten this to "Schizosaccharomyces specific, uncharacterized"?

**Comment by kimrutherford** [comment]
> I think that would be fine.

OK, that's live now.

I'll have a go at the taxonomic distribution later this week.

This is quite complicated:

((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) NOT ((conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) AND conserved in metazoa (PBO:0011069)))

I think it could be simplified to:

(conserved in archaea (PBO:0011072) OR conserved in bacteria (PBO:0011067)) NOT (conserved in metazoa (PBO:0011069))

**Comment by ValWood** [comment]
Yup, sorry. I didn't  look at the query!

**Comment by kimrutherford** [comment]
> Label :Eukaryotic, to Metazoa
> Label : Kingdom wide, to metazoa

Do these labels make sense?  They aren't clear to me.

**Comment by ValWood** [comment]
No they are rubbish!

suggest
 :Eukaryotes only, Fungi and Metazoa -> 
Kingdom wide, to metazoa -> Eukaryotes and prokaryotes

? better?

**Comment by kimrutherford** [comment]
Tweeted!
